cryptosuite/jsonwebkey2020.go

```diff
@@ -21,7 +21,7 @@ type (
 )
 
 const (
-	JsonWebKey2020 LDKeyType = "JsonWebKey2020"
+	JSONWebKey2020Name LDKeyType = "JsonWebKey2020"
```

Comment:
instead of name, maybe `JSONWebKey2020Type` ?

Reply:
I like that, done!